test/framework/resources/templates/logger-app.yaml.tpl

```diff
@@ -51,7 +51,20 @@ spec:
 {{ else }}
           value: 0s
 {{- end }}
+        resources:
+          limits:
+            cpu: 8m
+            memory: 20Mi
+          requests:
+            cpu: 4m
+            memory: 3Mi
```

Comment:
3Mi of memory is so low that even a container cannot be started by the runtime. I guess you can increase this a bit, isn't you?

Reply:
I was wrong, you have limit which is high enough, so everything is good.

Reply:
But that is about VPA that can scale down too much the container resources. Here, we have a deployment with static requests and limits. Yes, the requests are quite low, but the limits are still high enough to not let that issue with container runtime unable to create a container.